Lib/test/test_math.py

```diff
@@ -501,6 +501,16 @@ def testExp(self):
         self.assertTrue(math.isnan(math.exp(NAN)))
         self.assertRaises(OverflowError, math.exp, 1000000)
 
+    def testExp2(self):
+        self.assertRaises(TypeError, math.exp2)
+        self.ftest('exp2(-1)', math.exp2(-1), 0.5)
+        self.ftest('exp2(0)', math.exp2(0), 1)
+        self.ftest('exp2(1)', math.exp2(1), 2)
```

Comment:
Can we add at least one test for a non-integral finite input? E.g., 
```suggestion
        self.ftest('exp2(1)', math.exp2(1), 2)
        self.ftest('exp2(2.3)', math.exp2(2.3), 4.924577653379665)
```

Reply:
Done. It might also be a good idea to add non-integral testcases to other methods, like math.exp, but that's a separate issue